### DISCUSSION TITLE
Automatic trigger action after a few seconds

### DISCUSSION THREAD

**Discussion by MrMusaev** [filamentphp/filament#12195] [Help] [answered]
### Package

Panel builder

### Package Version

v3.2.61

### How can we help you?

I would like to trigger action (with modal) after a few seconds of the users open a page and show them a modal. Could you suggest any ways of achieving it? Thank you

**Comment by leandrocfe** [comment, marked as answer]
If you are using a custom page for example

```php
protected function getHeaderActions(): array
{
    return [
        Action::make('myAction')
            ->form([...]),
    ];
}
```

```html
<!-- resources/views/filament/pages/custom-page.blade.php-->
@script
    <script>
        setTimeout(() => {
            $wire.mountAction('myAction');
        }, 5000);
    </script>
@endscript
```

**Reply by MrMusaev** [reply]
Hi. Thank you for your reply. 
Is there any way to register the script in ViewPage or any other built-in pages? I did a little search of it, but couldn't find anything. But it seems must-have thing for me

**Reply by MrMusaev** [reply]
Thank you once again. I managed to do it. I created a new blade file and copied the initial content from the default filament one, then added the script you provided.
